x/auth/ante/setup.go

```diff
@@ -39,6 +39,14 @@ func (sud SetUpContextDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate
 
 	newCtx = SetGasMeter(simulate, ctx, gasTx.GetGas())
 
+	if cp := ctx.ConsensusParams(); cp != nil && cp.Block != nil {
```

Comment:
This differs from the original PR (https://github.com/cosmos/cosmos-sdk/pull/16547/)